homeassistant/components/lidarr/sensor.py

```diff
@@ -0,0 +1,168 @@
+        native_unit_of_measurement="Status",
+        icon="mdi:information",
+        value=lambda coordinator, _: coordinator.system_status.version,
+    ),
+    LidarrSensorEntityDescription(
+        key="wanted",
+        name="Wanted",
+        native_unit_of_measurement="Albums",
+        icon="mdi:music",
+        value=lambda coordinator, _: coordinator.wanted.totalRecords,
+    ),
+)
+
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Lidarr sensors based on a config entry."""
+    coordinator: LidarrDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    entities = []
+    for description in SENSOR_TYPES:
+        if description.key == "diskspace":
+            for mount in coordinator.disk_space:
+                desc = deepcopy(description)
+                name = mount.path.rsplit("/")[-1].rsplit("\\")[-1]
+                desc.key = f"{description.key}_{name}"
+                desc.name = f"{description.name} {name.capitalize()}"
+                entities.append(LidarrSensor(coordinator, desc, name))
+        else:
+            entities.append(LidarrSensor(coordinator, description))
+    async_add_entities(entities)
+
+
+class LidarrSensor(LidarrEntity, SensorEntity):
+    """Implementation of the Lidarr sensor."""
+
+    entity_description: LidarrSensorEntityDescription
+
+    def __init__(
+        self,
+        coordinator: LidarrDataUpdateCoordinator,
+        description: LidarrSensorEntityDescription,
+        ext_name: str = "",
+    ) -> None:
+        """Create Lidarr entity."""
+        super().__init__(coordinator)
+        self.entity_description = description
+        self._attr_name = f"{DEFAULT_NAME} {description.name}"
+        self._attr_unique_id = f"{coordinator.config_entry.entry_id}_{description.key}"
+        self.ext_name = ext_name
+
+    @property
+    def extra_state_attributes(self) -> dict[str, StateType | datetime] | None:
+        """Return the state attributes of the sensor."""
+        if self.entity_description.key == "queue":
+            return {i.title: queue_str(i) for i in self.coordinator.queue.records}
+        if self.entity_description.key == "status":
+            return self.coordinator.system_status.attributes
+        if self.entity_description.key == "wanted":
+            return to_attr(self.coordinator.wanted.records)
+        return None
+
+    @property
+    def native_value(self) -> StateType:
+        """Return the state of the sensor."""
+        return self.entity_description.value(self.coordinator, self.ext_name)
+
+
+def get_space(coordinator: LidarrDataUpdateCoordinator, name: str) -> str:
+    """Get space."""
+    space = [
+        mount.freeSpace / 1024 ** BYTE_SIZES.index(DATA_GIGABYTES)
+        for mount in coordinator.disk_space
+        if name in mount.path
+    ]
+    return f"{space[0]:.2f}"
+
+
+def to_attr(
```

Comment:
Let's remove this method because we really shouldn't store this much in an attribute. It's frequently changing too, causes a lot of disk writes.